ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -26,18 +26,13 @@
 AgentGroup = str
 
 
-class StepResult(NamedTuple):
+class DecisionRequest(NamedTuple):
```

Comment:
Personally I like the old name better

Reply:
I kind of prefer the new one because StepResult no longer contains "all" of the result of a step as it does not contain information about the dead agents. I also want to make it clear that the agents in this namedtuple are expecting an action from `env.set_actions`